apps/codeforafrica/src/components/ErrorHero/ErrorHero.js

```diff
@@ -0,0 +1,53 @@
+import { RichTypography, Section } from "@commons-ui/core";
+import React from "react";
+
+import TwoToneBackground from "@/codeforafrica/components/TwoToneBackground";
+
+const ErrorHero = React.forwardRef(function ErrorHero(props, ref) {
+  const { title, subtitle, sx, tags, ...other } = props;
+
+  return (
+    <TwoToneBackground
+      sx={{ py: { xs: 7.5, md: 10, lg: 12.5 } }}
+      {...other}
+      ref={ref}
+    >
+      <Section
+        sx={{
+          maxWidth: {
+            sm: "648px",
+            md: "912px",
+          },
+          px: { xs: 2.5, sm: 0 },
+          zIndex: 1,
+          ...sx,
+        }}
+      >
+        <RichTypography
+          sx={{
+            paddingBottom: 2.5,
+            fontSize: "48px",
```

Comment:
Why are we setting font-size manually? Just update `variant`/`typography`. For example: https://github.com/CodeForAfrica/ui/blob/009a9dc72d4e0236c38b9711b1caba46de39fdad/apps/codeforafrica/src/components/NavBarNavList/NavBarNavList.js#L25